kubernetes/annotations_and_labels.md

```diff
@@ -34,7 +34,7 @@ This page defines common annotations and labels we set in Kubernetes objects.
   `giantswarm.io/organization=dev`.
 - `giantswarm.io/randomkey` - value should contain randomkey name as defined in
   github.com/giantswarm/randomkeys repo. This is used in encryption Secrets.
-- `giantswarm.io/reconciled-version` - value should contain version reconciled by the operator (formerly known as versionbundle version).
+- `giantswarm.io/version` - value should contain version reconciled by the operator (formerly known as versionbundle version).
```

Comment:
Actually this should be removed. We have `OPERATOR.giantswarm.io/version` defined below and this is what is used.